client/base_client.go

```diff
@@ -216,7 +216,7 @@ func (c *baseClient) getMembers(ctx context.Context, url string) (*pdpb.GetMembe
 	members, err := pdpb.NewPDClient(cc).GetMembers(ctx, &pdpb.GetMembersRequest{})
 	if err != nil {
 		attachErr := errors.Errorf("error:%s target:%s status:%s", err, cc.Target(), cc.GetState().String())
-		return nil, errors.WithStack(attachErr)
+		return nil, errs.ErrClientGetMember.GenWithStackByArgs(attachErr)
```

Comment:
Should we change it to `errs.ErrClientGetMember.Wrap(errors.WithStack(attachErr)).FastGenWithCause()`?

Reply:
I think `errs.ErrClientGetMember.GenWithStackByArgs(attachErr)` is better